Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "clam"
-version = "0.3.2"
+version = "0.3.5"
```

Comment:
make sure you address this before merging it in. Whatever goes first should be `0.3.3`